### PR TITLE
Preventing duplicate registration and related unit test

### PR DESCRIPTION
**Pull request by jworcestBAH**
@nmonga91 or @bsweger:  This prevents users from repeating the registration process.

### CODE REVIEW

File: dataactbroker/handlers/accountHandler.py [hunk 1]
```diff
@@ -164,6 +164,12 @@ def ThreadedFunction (from_email="",username="",title="",agency="",userEmail=""
             return JsonResponse.error(exc,exc.status)
         # Find user that matches specified email
         user = self.interfaces.userDb.getUserByEmail(requestFields.getValue("email"))
+        # Check that user's status is before submission of registration
+        print("About to register a user with current status of " + str(user.user_status_id))
```
Comment (bsweger) [inline dataactbroker/handlers/accountHandler.py]: Is this a debug statement that should come out?

**Comment by bsweger** [comment]
@jworcestBAH done reviewing--just one quick question about a print statement. Otherwise looks good.

File: dataactbroker/handlers/accountHandler.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactbroker/handlers/accountHandler.py]: Yes, missed that one.  Fixing.

**Comment by bsweger** [comment]
Looks good--merging!

**Pull request merged**